src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -270,6 +271,8 @@ class ContentSidebar extends React.Component<Props, State> {
      * @return {void}
      */
     fetchFileSuccessCallback = (file: BoxItem): void => {
+        const { onFetchFileSuccess }: Props = this.props;
+        onFetchFileSuccess && onFetchFileSuccess();
```

Comment:
I think if possible its safer to double coalesce `!!onFetchFileSuccess` since it's not a boolean. I've ran into some weird cases when using the `&&` with a function call. Might also be more readable with a conditional instead